index.go

```diff
@@ -110,7 +111,7 @@ func NewIndex(metricsEndpoint *string, ocpMetaAgent *ocpmetadata.Metadata) *cobr
 			for _, prometheusClient := range metricsScraper.PrometheusClients {
 				prometheusJob := prometheus.Job{
 					Start: time.Unix(start, 0),
-					End:   time.Unix(end, 0),
+					End:   time.Unix(end+600, 0),
```

Comment:
Moving logic to capture [last 10 minutes metrics](https://github.com/cloud-bulldozer/e2e-benchmarking/blob/master/workloads/kube-burner-ocp-wrapper/run.sh#L127C33-L127C36) here itself so that we can have enough flexibility in CI and e2e to pass the actual timestamps. Doing the same for [workers-scale](https://github.com/kube-burner/kube-burner-ocp/pull/136/files#diff-36b18e1f4d646364265db6b064284941c083699ca5551ed1647f11c0d4e985c5R157) as well.